docs/react/your-first-app/4-loading-photos.md

```diff
@@ -4,62 +4,199 @@ sidebar_label: Loading Photos
 ---
 
 <head>
-  <title>Loading Photos from the Filesystem Using A Key-Value Store</title>
+  <title>Loading Photos from the Filesystem with React | Ionic Capacitor Camera</title>
   <meta
     name="description"
     content="We’ve implemented photo taking and saving to the filesystem, now learn how Ionic leverages Capacitor Preferences API for loading our photos in a key-value store."
   />
 </head>
 
+# Loading Photos from the Filesystem
```

Comment:
We can move this to the frontmatter as `title`.

Reply:
https://github.com/ionic-team/ionic-docs/pull/4201/commits/6a96994c668a58ebecb35605f182003070d95285